### ISSUE TITLE
REZ model

### ISSUE THREAD

**Issue by hsywu2020** [MikeJSeo/bnma#3]
Hi:

I am just wondering whether the package can deal with 'REZ' model, if it can, are there any example code to play around? I attached the definition of REZ model in the following:

'REZ' model: allowing treatments to vary in efficacy across pasi cut-offs

Thank you very much

<img width="492" alt="Image" src="https://github.com/user-attachments/assets/e36728a4-72e2-4d0b-a046-5a24e806e51f" />

**Comment by MikeJSeo** [comment]
Hi,
I don't think they are the same.
Currently, model uses baseline category logits model described in this [paper](https://onlinelibrary.wiley.com/doi/epdf/10.1002/jrsm.1103).

![Image](https://github.com/user-attachments/assets/5593e6a2-a331-46f7-94c9-d83a85c60936)

Best,
Michael

**Comment by hsywu2020** [comment]
Sorry, I am not sure I understand. I know bnma can handle baseline risk adjustment for multinomial outcomes. My question is whether our bnma package can also replicate REZ model as I attached. does REZ model related to two models you sent here?
Thank you

**Comment by MikeJSeo** [comment]
No, two models look quite different unfortunately
